### DISCUSSION TITLE
Please Add SwiftUI style guide for Swift World 🚀

### DISCUSSION THREAD

**Discussion by keshavvishwkarma** [airbnb/swift#186] [General]
A style guide for the SwiftUI domain-specific language would be a nice addition to Airbnb's Swift style guide for the Swift World 🚀

**Comment by fdiaz** [comment]
Hi @keshavvishwkarma ! Thanks for participating in our style guide repo discussions :wink:
​
I personally like the idea of having style recommendations on SwiftUI usage so we can move towards having best practices in the community, but given that:
1. SwiftUI is still an early stage technology
2. We (Airbnb) don't currently use SwiftUI in production

I think we won't move in this direction for the time being.
​
When / if we start using SwiftUI in production for a while and we've seen how both the Airbnb iOS community and the larger iOS community think of style in SwiftUI I think we should definitively start adding recommendations in that direction.
​
In the short term we've been quite happy with using [Epoxy](https://github.com/airbnb/epoxy-ios/wiki/FAQ#why-would-i-use-epoxy-and-uikit-instead-of-swiftui) for our declarative UI, so I don't expect changes on this for now.
